libs/libc/debug/lib_backtrace.c

```diff
@@ -44,7 +44,7 @@ struct trace_arg
  * Private Functions
  ****************************************************************************/
 
-static inline weak_function
+weak_function
```

Comment:
Could you at least move this change to a separate commit? This is not related to the above change.

Reply:
Done, thanks.